### ISSUE TITLE
bbl destroy not working for version 7.2, but works for 6.10

### ISSUE THREAD

**Issue by cortega-lh** [cloudfoundry/bosh-bootloader#447]
I'm having issues installing running `bbl up` on AWS due an issue with the terraform version (see https://github.com/cloudfoundry/bosh-bootloader/issues/445). 

After I saw the error message, I tried to revert all changes running `bbl down`, but I receive an error message that says `jumpbox-vars-store.yml: no such file or directory`. 

The version I used is `bbl 7.2.12 (darwin/amd64)`.

After downgrading my bbl version to  `bbl 6.10.54 (darwin/amd64)` I was able to revert those changes.

I don't know it this is helpful for you or not, but I found a similar issue (see https://github.com/cloudfoundry/bosh-bootloader/issues/121) it's closed and it has no details about how this issue was solved.

**Comment by zachgersh** [comment]
@cortega-lh - appreciate the report. I would have been interested to see if `bbl destroy` would have actually done the right thing. I know they are supposed to be aliases but I've seen oddities in the past.

Could you try it again potentially?

you can also see our test version here:

https://github.com/cloudfoundry/bosh-bootloader/blob/master/acceptance-tests/bbl/upgrade_test.go#L45

**Comment by gerhard** [comment]
I've hit the same exact issue with v7.2.12 & v7.3.5:

```
gmake bbl_destroy -n
/Users/gerhard/github.com/rabbitmq/iaas/bin/bbl-v7.2.12_osx --debug destroy \
  --iaas gcp --gcp-region europe-west4 \
  --gcp-service-account-key /Users/gerhard/github.com/rabbitmq/iaas/bosh-bootloader-20190304.key.json

gmake bbl_destroy
Are you sure you want to delete infrastructure for "rabbitmq"? This operation cannot be undone! (y/N): y
step: cleaning up director resources
Read jumpbox vars file: open /Users/gerhard/github.com/rabbitmq/iaas/vars/jumpbox-vars-store.yml: no such file or directory
gmake: *** [/Users/gerhard/github.com/rabbitmq/iaas/mk/bosh.mk:123: bbl_destroy] Error 1
```

Downgrading to v6.10.54 and trying again.

It would have been nice if `terraform destroy` worked - what am I doing wrong?

```
terraform plan -state=../vars/terraform.tfstate -var-file=../vars/bbl.tfvars
var.credentials
  Enter a value: ^C
Interrupt received.
Please wait for Terraform to exit or data loss may occur.
Gracefully shutting down...

Error: Error asking for user input: Error asking for credentials: interrupted
```

**Comment by gerhard** [comment]
I can confirm that v6.10.54 works, no luck with v7.x:

```
gmake bbl_destroy -n
/Users/gerhard/github.com/rabbitmq/iaas/bin/bbl-v6.10.54_osx --debug destroy \
  --iaas gcp --gcp-region europe-west4 \
  --gcp-service-account-key /Users/gerhard/github.com/rabbitmq/iaas/bosh-bootloader-20190304.key.json

gmake bbl_destroy
Are you sure you want to delete infrastructure for "rabbitmq"? This operation cannot be undone! (y/N): y
step: generating terraform template
step: generating terraform variables
step: terraform init
... # elided for brevity
Destroy complete! Resources: 25 destroyed.
step: finished destroying infrastructure
```

**Comment by gerhard** [comment]
Does anyone know if this works in the recently released v8.0.0?
